marathon/mesos-shuffle-service.json

```diff
@@ -22,7 +22,7 @@
   "container": {
     "type": "DOCKER",
     "docker": {
-      "image": "registry.marathon.mesos:5000/spark:2.7.0-2.4.0-hadoop-2.7-scala-2.12",
+      "image": "mesosphere/spark:2.7.0-2.4.0-hadoop-2.7",
```

Comment:
Is this scala 2.12 or 2.11? I think spark moved to 2.12 from 2.4.1 onwards (https://spark.apache.org/docs/2.4.0/index.html). I don't know if it matters.

Reply:
Scala 2.11 and yes it does matter :)

Reply:
The branch name is wrong